tests/azure/resourcegroup_test.go

```diff
@@ -20,19 +21,23 @@ func testResourceGroupCreate(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 
+	logger := log.New(t, "testResourceGroupCreate")
+	defer logger.Close()
```

Comment:
Just to close the file, don't know how to avoid the abstraction from leaking and not leak an open file :-). I could call it FileLogger so it gets more clear that it is a file and the Close thing is necessary.